src/pages/ThorChainLP/components/AddLiquidity/AddLiquidityInput.tsx

```diff
@@ -263,6 +271,9 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
   const pool = useMemo(() => pools?.find(pool => pool.assetId === assetId), [assetId, pools])
 
   const _poolAsset = useAppSelector(state => selectAssetById(state, assetId ?? ''))
+  const feeAsset = useAppSelector(state =>
```

Comment:
we have `poolAssetFeeAsset`

Reply:
d2428a405fc4d0ab4a7534bcf545dd150ff1e13c